src/collective/lazysizes/upgrades/v2/configure.zcml

```diff
@@ -25,6 +25,13 @@
         handler="..cook_javascript_resources"
         />
 
+    <genericsetup:upgradeDepends
```

Comment:
@rodfersou @winstonf88 @idgserpro this is an alternative way of running an upgrade step that I just learned with @pcdummy.

in this case, we are re-importing `registry.xml` without any Python code.